client/my-sites/upgrades/checkout-thank-you/footer.jsx

```diff
@@ -51,13 +69,27 @@ const CheckoutThankYouFooter = React.createClass( {
 		return (
 			<Card className="checkout-thank-you__footer">
 				<div className="checkout-thank-you__footer-content">
+					{ this.renderGravatar() }
```

Comment:
This has become a nice component. Can we turn it into an app-component we can reuse outside of thank you pages? Thinking something like `SupportFromHappiness` that we can render in `devdocs/app-components`, on the `/plans` page, etc.

(It would also help clean up the thank-you stylesheet.)